src/flash/com/longtailvideo/jwplayer/player/Player.as

```diff
@@ -62,13 +56,23 @@ public class Player extends Sprite implements IPlayer {
         this.removeEventListener(Event.ADDED_TO_STAGE, stageReady);
         RootReference.init(this);
         _view.setupView();
+
+        // trigger throttling in Chrome before additional setup or loading occurs
+        this.addEventListener('throttle', flashThrottled);
+        _view.drawKeyframe();
     }
 
-    private function flashThrottled(e:ThrottleEvent):void {
+    private function flashThrottled(e:Event):void {

```

Comment:
Replaced the import of ThrottleEvent and ThrottleType classes / constants with String literals to avoid compile issues targeting Flash Player 11.2.